pkgs/development/compilers/zig/0.11.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, llvmPackages
+, libxml2
+, zlib
+, coreutils
+}@args:
+
+import ./generic.nix args "sha256-iuU1fzkbJxI+0N1PiLQM013Pd1bzrgqkbIyTxo5gB2I=" {
```

Comment:
Why not to use hash as a named argument instead of a magic constant?

Reply:
do you mean as part of `args` or the big attrset? this is so I don't have to use things like `removeAttrs`

Reply:
It makes the expression less clear. Is there any serious drawback on `removeAttrs`, like a mass rebuild?

Reply:
> Is there any serious drawback on `removeAttrs`, like a mass rebuild?

No, it just takes a little more code to implement, I will switch to that